### ISSUE TITLE
Appendix B.1 and B.2 need some TLC

### ISSUE THREAD

**Issue by thomas-fossati** [lamps-wg/csr-attestation#155]
@henkbirkholz asked me to look at B.1 and B.2.

Two observations:

* The title is the same and a good percentage of the prose portion is repeated. It is not clear why both sections are needed.  (My guess is you wanted to have CMW and TCBInfo examples one alongside the other, but for some reason, you ended in this garbled situation.)
* The example in B.2 is not a CMW, it's a DICE TCBInfo.  To be a CMW it should use OID [id-pe-cmw](https://www.iana.org/assignments/smi-numbers/smi-numbers.xhtml#smi-numbers-1.3.6.1.5.5.7.1), and the encoding defined in [Section 5 of CMW](https://www.ietf.org/archive/id/draft-ietf-rats-msg-wrap-08.html#section-5-8).

In conclusion:
1. The title of B.2 should be "TCG DiceTCBInfo in CSR", and
2. B.1 still needs some tender love and care.
 
This may be related to #144

**Comment by henkbirkholz** [comment]
[6.4.2. Initial Registry Contents](https://lamps-wg.github.io/csr-attestation/draft-ietf-lamps-csr-attestation.html#name-initial-registry-contents) references [I-D.ietf-rats-msg-wrap](https://lamps-wg.github.io/csr-attestation/draft-ietf-lamps-csr-attestation.html#I-D.ietf-rats-msg-wrap) (oid [1.3.6.1.5.5.7.1.35](https://www.iana.org/assignments/smi-numbers/smi-numbers.xhtml#smi-numbers-1.3.6.1.5.5.7.1.35)) and then lists DiceConceptualMessageWrapper (oid 2.23.133.5.4.9) as the item to register.

1.3.6.1.5.5.7.1.35 represents a superset of 2.23.133.5.4.9, I think. Should we consider using that oid instead?

**Comment by thomas-fossati** [comment]
> 1.3.6.1.5.5.7.1.35 represents a superset of 2.23.133.5.4.9, I think. Should we consider using that oid instead?

Yes, two reasons:

1. As you also noted, IETF CMW is a superset of TCG CMW;
2. It makes sense for an IETF spec to eat its own dog food :-)

**Comment by nedmsmith** [comment]
Here's proposed ASN.1 for Section B.1
I noted there was no name given to the DEFINITIONS section that defined EvidenceStatementSet. Was there a reason?
I found it helpful to give it a name so I can write the IMPORTS section, namely: EvidenceStatementSet FROM CsrAttestation 

Should I give it a name in a PR to add the other ASN.1?

Note also the TcgDice block defines the other DICE OIDs that show up in the OID registry in the IANA section.

```
CsrAttestationExample DEFINITIONS IMPLICIT TAGS ::= BEGIN

IMPORTS 

tcg-dice-conceptual-message-wrapper FROM TcgDice
DiceConceptualMessageWrapper FROM TcgDice

EvidenceStatementSet FROM CsrAttestation 
;

tcgDiceEvidenceStatementES EVIDENCE-STATEMENT ::=
  { DiceConceptualMessageWrapper IDENTIFIED BY tcg-dice-conceptual-message-wrapper }

-- where ConceptualMessageWrapper and tcg-dice-conceptual-message-wrapper
-- are defined in DICE-Attestation-Architecture-Version-1.1-Revision-17_1August2023.pdf

EvidenceStatementSet EVIDENCE-STATEMENT ::= {
  tcgDiceEvidenceStatementES, ...
}
END

TcgDice DEFINITIONS IMPLICIT TAGS ::= BEGIN

EXPORTS ALL;

tcg OBJECT IDENTIFIER ::= { 2 23 133 }
tcg-dice OBJECT IDENTIFIER ::= { tcg platformClass(5) dice(4) }
tcg-dice-TcbInfo OBJECT IDENTIFIER ::= { tcg-dice tcbinfo(1) }
tcg-dice-MultiTcbInfo OBJECT IDENTIFIER ::= {tcg-dice multitcbinfo(5) }
tcg-dice-UCCS-evidence OBJECT IDENTIFIER ::= {tcg-dice uccs-evidence(6) }
tcg-dice-manifest-evidence OBJECT IDENTIFIER ::= {tcg-dice manifest-evidience(7) }
tcg-dice-MultiTcbInfoComp OBJECT IDENTIFIER ::= {tcg-dice multitcbinfocomp(8) }
tcg-dice-conceptual-message-wrapper OBJECT IDENTIFIER ::= { tcg-dice cmw(9) }

DiceConceptualMessageWrapper ::= SEQUENCE {
    cmw OCTECT STRING
}
END
```

**Comment by nedmsmith** [comment]
Fixed in PR #160

**Comment by hannestschofenig** [comment]
Close based on the PR from Ned.
